src/ui/public/assets/favicons/safari-pinned-tab.svg

```diff
@@ -2,18 +2,93 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
  "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
 <svg version="1.0" xmlns="http://www.w3.org/2000/svg"
- width="260.000000pt" height="260.000000pt" viewBox="0 0 260.000000 260.000000"
+ width="261.000000pt" height="261.000000pt" viewBox="0 0 261.000000 261.000000"
```

Comment:
why do we need this increment in size?

Reply:
This icon are generated online, from favicon generator, I get it from there